CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+- Adds `serviceAccount` option to `runtimeOptions`, which lets users to specify what service account a Cloud Function should run as. For example: ```
```

Comment:
I don't think inline ``` is legal

Reply:
Moved this to its own line, I think this should work now.